### PR TITLE
add Memesena Presence

### PR DESCRIPTION
**Pull request by AyamDobhal**
added a presence for [Memesena](https://www.memesena.com)
kindly review and merge if found correct. Can provide working proof if required.

### CODE REVIEW

File: websites/M/Memesena/presence.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
+var presence = new Presence( {
```
Comment (ririxichan) [inline websites/M/Memesena/presence.ts:1]: var -> const

File: websites/M/Memesena/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "name": "AyamDobhal",
+      "id": "379269487532310530"
+    },
+
+    "service": "Memesena",
+    "description": {
+      "en": "The perfect place for memers where we have Desi and English meme templates, meme articles, meme arts, copypastas, emojipastas and meme bots."
+    },
+    "url": "memesena.com",
+    "version": "1.0",
```
Comment (ririxichan) [inline websites/M/Memesena/dist/metadata.json:12]: 1.0.0

**Review by ririxichan — CHANGES_REQUESTED** [review]
Proof is required and fix these issues.

**Comment by AyamDobhal** [comment]
> Proof is required and fix these issues.

alright I'll fix them in a few mins. What kind of proof do I need to submit? Screenshots of presence working?

**Comment by ririxichan** [comment]
yes, proof of working presence

**Comment by AyamDobhal** [comment]
Made the changes as requested in both the files. Attaching proof of working below.
image
image
image
image

Hope everything is fine now.

File: websites/M/Memesena/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+      "name": "AyamDobhal",
+      "id": "379269487532310530"
+    },
+
+    "service": "Memesena",
```
Comment (Bas950) [inline websites/M/Memesena/dist/metadata.json:7]: remove that empty line

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Pull request merged**